lib/scavenger_hunt/app/views/layouts/scavenger_hunt/_location_styles.html.slim

```diff
@@ -27,6 +27,12 @@ css:
     border-color: #{color};
   }
 
+  input:focus {
```

Comment:
Please also account for `textarea` and `select` when styling form elements!

Reply:
@flipsasser What do you mean by that? I think I get that you want this style to be like: `input:focus, textarea:focus, select:focus`? But I want to make sure I understand the why here. Is it just to make it all consistent? Would I always want all the same styles applied to all of those elements?